airbyte-integrations/connectors/destination-s3/src/test/resources/parquet/json_schema_converter/get_field_type.json

```diff
@@ -0,0 +1,93 @@
+[
```

Comment:
this seems to be algorithmically generated, would it make things easier to read if we put it in the test case itself and generate it in code? it also makes it closer to the test code itself

Reply:
No, this is manually generated. I manually wrote the Avro schema from the Json schema. I am not aware of anything that can generate both schemas from some definition.